src/storybook/stand-alone-documentaion/shadow/card-shadow-example/card-shadow-example.jsx

```diff
@@ -0,0 +1,23 @@
+import { Frame } from "../../../components";
+import { BEMClass } from "../../../../helpers/bem-helper";
+import { Button, Heading } from "../../../../components";
+import { peopleColumn } from "../assets";
+import "./card-shadow-example.scss";
+
+const CSS_BASE_CLASS = "monday-storybook-card-shadow-example";
+const bemHelper = BEMClass(CSS_BASE_CLASS);
```

Comment:
we have css modules - lets prefer them instead of BEM as we need to move away from bem